modules/monitoring/application/forms/Command/Object/ScheduleServiceDowntimeCommandForm.php

```diff
@@ -68,37 +99,41 @@ public function createElements(array $formData = array())
                         . ' the host or service that is having problems. Make sure you enter a brief description of'
                         . ' what you are doing.'
                     ),
-                    'attribs'       => array('class' => 'autofocus')
+                    'attribs'       => array('class' => 'autofocus'),
+                    'value'         => $this->commentText
                 )
             ),
             array(
                 'dateTimePicker',
                 'start',
                 array(
-                    'required'      => true,
-                    'label'         => $this->translate('Start Time'),
-                    'description'   => $this->translate('Set the start date and time for the downtime.'),
-                    'value'         => $start
+                    'required'        => true,
+                    'label'           => $this->translate('Start Time'),
+                    'description'     => $this->translate('Set the start date and time for the downtime.'),
+                    'preserveDefault' => true,
+                    'value'           => $this->start
                 )
             ),
             array(
                 'dateTimePicker',
                 'end',
                 array(
-                    'required'      => true,
-                    'label'         => $this->translate('End Time'),
-                    'description'   => $this->translate('Set the end date and time for the downtime.'),
-                    'value'         => $end
+                    'required'        => true,
+                    'label'           => $this->translate('End Time'),
+                    'description'     => $this->translate('Set the end date and time for the downtime.'),
+                    'preserveDefault' => true,
```

Comment:
That's the only time `preserveDefault` is required, isn't it? Please remove the other usages.